example/controller-registration.yaml

```
@@ -4,11 +4,13 @@ kind: ControllerRegistration
 metadata:
   name: kupid
 spec:
+  resources:
+  - kind: ClusterPorSchedulingPolicy
+    type: 
```

```suggestion
  resources:
  - kind: ClusterPodSchedulingPolicy
    type: 
```

Do you need this? If so, technically, `PodSchedulingPolicy` is also applicable here.